sdk/agent/agent.go

```diff
@@ -250,6 +250,10 @@ var handlerMap = map[msg.Type]func(*Agent, msg.Message, *msg.Encoder) error{
 	msg.TypeCloseResponse:   (*Agent).handleCloseResponse,
 }
 
+func isInitiator(self, other *keypair.FromAddress) bool {
+	return self.Address() > other.Address()
```

Comment:
❓ is this just saying whichever address is higher in the lexical order is the initiator? Is that just a simplification for now?

Reply:
> is this just saying whichever address is higher in the lexical order is the initiator?

Yup.

> Is that just a simplification for now?

Yes, but it might work long term. It doesn't make any difference to the participants who the initiator is. We need to pick one to be and this seems simple.